hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaDeploymentManagement.java

```diff
@@ -286,7 +284,9 @@ private DistributionSetAssignmentResult assignDistributionSetToTargets(@NotNull
     private void sendDistributionSetAssignmentEvent(final List<JpaTarget> targets, final Set<Long> targetIdsCancellList,
```

Comment:
Due the method is now also sending `sendTargetUpdatedEvent` I would probably re-name the method from `sendDistributionSetAssignmentEvent` to `sendAssignmentEvents` or something because it's not only a `DistributionSetAssignmentEvent` anymore.